vulns.go

```diff
@@ -46,18 +47,43 @@ func getVulnsWithoutTicket(flags flags, projectID string, maturityFilter []strin
 	if flags.optionalFlags.issueType != "all" && flags.optionalFlags.issueType != "" {
 		body.Filters.Types = []string{flags.optionalFlags.issueType}
 	}
-	switch flags.optionalFlags.severity {
-	case "critical":
-		body.Filters.Severities = []string{"critical"}
-	case "high":
-		body.Filters.Severities = []string{"critical", "high"}
-	case "medium":
-		body.Filters.Severities = []string{"critical", "high", "medium"}
-	case "low":
-		body.Filters.Severities = []string{"critical", "high", "medium", "low"}
-	default:
-		log.Fatalln("Unexpected severity threshold")
-	}
+
+    if len(flags.optionalFlags.severities) > 0 && len(flags.optionalFlags.severityThreshold) == 0 {
```

Comment:
@Ipuvi should we extract out this logic and call it getSeverties or something, this looks like being duplicated in both the methods

Reply:
Done